mad-dog/src/tests/test_userReplicaSetNodes/index.js

```diff
@@ -1,17 +1,18 @@
 const { addAndUpgradeUsers } = require('../../helpers.js')
 const verifyValidCNs = require('./verifyValidCN')
-// const deregisterRandomCreatorNode = require('./deregisterRandomCreatorNode.js')
+const deregisterRandomCreatorNode = require('./deregisterRandomCreatorNode.js')
 const stopRandomCreatorNode = require('./stopRandomCreatorNode.js')
 const setNumCreatorNodes = require('./setNumCreatorNodes.js')
 const { uploadTracksforUsers } = require('../../utils/uploadTracksForUsers')
+const { logger } = require('../../logger')
 
-const MAX_ATTEMPTS_TO_VALIDATE_REPLICA_SET = 10
+const MAX_ATTEMPTS_TO_VALIDATE_REPLICA_SET = 20
 const WAIT_INTERVAL_TO_UPDATE_REPLICA_SET_MS = 20000
```

Comment:
local dev snapback interval is 10seconds with 3 modulo bases (i recently changed), so might be worth setting this to a higher value but all good, in 20 attempts should 100000% resolve lol

Reply:
yeeeeeee hopefully 20 is sufficient 😬